migrations/20240916110123_create_items_table.js

```diff
@@ -0,0 +1,40 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('items', function (table) {
```

Comment:
```suggestion
  return knex.schema.createTable('entries', function (table) {
```

Let's change the table name to `entries`